.github/workflows/commitlint.yaml

```
@@ -4,7 +4,8 @@ on:
   # This workflow is triggered on pull requests to the main branch.
   pull_request:
     branches: [main]
-    types: [opened, edited, synchronize]
+    # milestoned is added here as a workaround for release-please not triggering PR workflows (PRs should be added to a milestone to trigger the workflow).
+    types: [milestoned, opened, reopened, synchronize]
```

```suggestion
    types: [milestoned, opened, edited, synchronize]
```
reopened doesn't matter as much as edited on this one...